tests/parser/functions/test_convert.py

```diff
@@ -294,7 +294,10 @@ def _py_convert(val, i_typ, o_typ):
         # Note: Decimal(True) == Decimal("1")
         return _convert_int_to_decimal(val, o_typ)
 
-    val_bits = _to_bits(val, i_typ)
+    try:
+        val_bits = _to_bits(val, i_typ)
+    except eth.codecs.abi.exceptions.EncodeError:
```

Comment:
why does this raise an exception now?

Reply:
This should be fixed now - the exception was not getting caught where it should have been because of a typo.